### PR TITLE
Two luks related fixes

### PR DESCRIPTION
**Pull request by vojtechtrefny**

### CODE REVIEW

File: blivetgui/list_partitions.py [hunk 1]
```diff
@@ -53,42 +53,34 @@ def update_partitions_list(self, selected_device):
 
         self.partitions_list.clear()
 
+        def _get_real_child(child):
+            """ For luks, vgs parents add the luks device or vg, not the parrtition """
```
Comment (vpodzime) [inline blivetgui/list_partitions.py]: This is very cryptic and hard to understand even for me.

File: blivetgui/list_partitions.py [hunk 2]
```diff
@@ -53,42 +53,34 @@ def update_partitions_list(self, selected_device):
...
+            if self._is_group_device(child):
+                return self.blivet_gui.client.remote_call("get_group_device", child)
+            elif child.format and child.format.type == "luks" and child.children:
+                return self.blivet_gui.client.remote_call("get_luks_device", child)
+            else:
+                return child
+
         def _add_chilren(childs, parent_iter=None):
```
Comment (vpodzime) [inline blivetgui/list_partitions.py]: *childs*? Wow :D

File: blivetgui/list_partitions.py (hunk 1 above)
Comment (vojtechtrefny) [inline blivetgui/list_partitions.py]: Added a better explanation.

File: blivetgui/list_partitions.py (hunk 2 above)
Comment (vojtechtrefny) [inline blivetgui/list_partitions.py]: "blivet-gui: proudly ignoring English grammar since 2014"

Comment (vpodzime) [inline blivetgui/list_partitions.py]: Fine with me :)

**Pull request merged**